src/modal/modal.module.ts

```diff
@@ -1,8 +1,19 @@
 import {NgModule} from '@angular/core';
 
+import {NgbModalContainer} from './modal_container';
```

Comment:
elswere in the project all the files are named with `-`, not `_`, ex. `ngb-time.ts`, `ngb-typeahead-window.ts`